ocp_install_env.sh

```diff
@@ -117,7 +117,7 @@ $(network_configuration)
     dnsVIP: ${DNS_VIP}
     hosts:
 $(node_map_to_install_config_hosts $NUM_MASTERS 0 master)
-$(node_map_to_install_config_hosts $NUM_WORKERS $NUM_MASTERS worker)
+$(node_map_to_install_config_hosts $NUM_VM_WORKERS $NUM_MASTERS worker)
```

Comment:
I tested and found one problem with this - all of the workers get registered via the install-config, even though the replicas are set to NUM_WORKERS - this can cause problems because if NUM_VM_WORKERS is large (I tried 20) all of the VMs try to start enrollment then introspection at the same time, and in my environement this causes issues with VBMC that leads to failures.

I think perhaps we should leave this set to NUM_WORKERS, and generate the BMH yaml so that can be applied after initial deployment?

Reply:
Note we did have code that did the BMH generation prior to https://github.com/openshift-metal3/dev-scripts/pull/897 so we could perhaps restore some of that to do what's described above.

Reply:
how is that different from setting NUM_WORKERS to 20?

Reply:
> how is that different from setting NUM_WORKERS to 20?

The difference is you could scale out in smaller increments and the hosts aren't all registered via the installer (otherwise you'd just use NUM_WORKERS, right?)